README.md

```diff
@@ -1,583 +1,53 @@
-# K8s Custom Resource and Operator For TensorFlow jobs
+# Kubernetes Operator For TensorFlow jobs
 
 [![Build Status](https://travis-ci.org/tensorflow/k8s.svg?branch=master)](https://travis-ci.org/tensorflow/k8s)
 [![Coverage Status](https://coveralls.io/repos/github/tensorflow/k8s/badge.svg?branch=master)](https://coveralls.io/github/tensorflow/k8s?branch=master)
 [![Go Report Card](https://goreportcard.com/badge/github.com/tensorflow/k8s)](https://goreportcard.com/report/github.com/tensorflow/k8s)
-[Prow Test Dashboard](https://k8s-testgrid.appspot.com/sig-big-data)
-[Prow Jobs](https://prow.k8s.io/?repo=tensorflow%2Fk8s)
+[![Prow Test Dashboard](https://img.shields.io/badge/prow-dashboard-blue.svg)](https://k8s-testgrid.appspot.com/sig-big-data)
+[![Prow Jobs](https://img.shields.io/badge/prow-status-blue.svg)](https://prow.k8s.io/?repo=tensorflow%2Fk8s)
+[![Argo](https://img.shields.io/badge/argo-dashboard-blue.svg)](http://testing-argo.kubeflow.io/timeline)
 
 ## Overview
 
-TFJob provides a Kubernetes custom resource that makes it easy to
+The operator, which is a part of [Kubeflow](https://github.com/kubeflow/kubeflow), provides a Kubernetes custom resource that makes it easy to
```

Comment:
How about: custom resource definition (TFJob)

Reply:
Do you mean s/`provides a Kubernetes custom resource`/`provides a Kubernetes custom resource definition(TFJob)`?